app/assets/javascripts/controllers/catalog/catalog_item_form_controller.js

```diff
@@ -69,6 +71,13 @@ ManageIQ.angular.app.controller('catalogItemFormController', ['$scope', 'catalog
     }
   };
 
+  var getRemoveResourcesTypes = function() {
+    if (angular.isUndefined(vm.catalogItemModel.retirement_playbook_id) || vm.catalogItemModel.retirement_playbook_id === '')
+      vm.catalogItemModel['remove_resources_types'] = {"No": "no", "Yes": "yes"};
```

Comment:
Does this defines the value passed to the API call? I think we agreed on the values, here will be "no_without_playbook" and "yes_without_playbook"